src/main/webapp/consentGroupReview/ConsentGroupReview.js

```diff
@@ -217,8 +218,12 @@ class ConsentGroupReview extends Component {
     );
   }
 
-  rejectConsentGroup = (e) => () => {
-
+  rejectConsentGroup() {
+    spinnerService.showAll();
+    ConsentGroup.rejectConsent(this.props.rejectConsentUrl, this.props.consentKey).then(resp => {
+      window.location.href = this.getRedirectUrl(this.props.projectKey);
+      spinnerService.hideAll();
```

Comment:
I didn't catch this before. Could there be an error case where the response fails and `spinnerService.hideAll();` is never invoked? Should this be in a separate handler block? I don't remember the syntax, but there should be an error case for this promise that you could invoke this in.